.gitignore

```diff
@@ -59,3 +65,6 @@ db.sqlite3
 # Site media
 /data/
 src/backend/joanie/static/joanie/
+
+# Translations
+*.mo
```

Comment:
Just revert this change.

Reply:
It's done in the fixup, it will do it automatically with the rebase https://github.com/openfun/joanie/pull/130/commits/6806a460b80217cc4b1e3ff8d562a6216311046c